modules/common/src/main/java/com/sygic/modules/common/di/component/ModulesComponent.java

```diff
@@ -0,0 +1,29 @@
+package com.sygic.modules.common.di.component;
```

Comment:
je to blbost, ale ked uz presuvas... ono componenty pouzivaju moduly, takze ak by som daval nejake delenie do balikov, tak mne to skor evokuje hierarchiu, ze modules su "nizsie" ako components a nie na rovnakej urovni. navyse modules bude viac, component, ktory ich pouziva 1

-- di
    -- modules
        module.java
   component.java

pripadne potom takto (to ale uz kus zbytocne hlboke)

-- di
    -- comonents
        component.java
        -- modules
            module.java